### PR TITLE
Schedule runs

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: scripts/dataset/main.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-import { release } from './index.js';
+import cron from 'croner';
```
Comment (martinratinaud) [inline scripts/dataset/main.js:1]: I imagine croner will not have the same problems as node-schedule but how can you be sure ?

**Review by martinratinaud — APPROVED** [review]

File: src/main.js [hunk 2]
```diff
@@ -61,9 +61,9 @@ const TRACK_CHANGES_HOURS_INTERVAL = 24;
   }
 
   logger.info('The scheduler is running…');
+  logger.info('Documents will be tracked at minute 30 past hour 03, 09, 15, and 21 every day');
 
-  logger.info(`Documents will be tracked every ${TRACK_CHANGES_HOURS_INTERVAL} hours\n`);
-  setInterval(async () => {
+  cron('30 3,9,15,21 * * *', async () => {
```
Comment (MattiSG) [inline src/main.js:66]: This sounds more complicated than necessary to me 🙂 

```suggestion
logger.info('Documents will be tracked every six hours starting at midnight');

  cron('0 0/6 * * *', async () => {
```

File: src/main.js [hunk 3]
```diff
@@ -61,9 +61,9 @@ const TRACK_CHANGES_HOURS_INTERVAL = 24;
...
     await archivist.trackChanges(serviceIds);
```
Comment (MattiSG) [inline src/main.js:67]: Why would we `await` this? 🤔 
```suggestion
    archivist.trackChanges(serviceIds);
```

**Review by MattiSG — APPROVED** [review]
I hope this one lib doesn't let us down 😄

But it does seem well documented.

**Comment by Hexagon** [comment]
> I hope this one lib doesn't let us down 😄
> 
> But it does seem well documented.

Should work! If it don't, just drop an issue and we'll sort it out ;)

**Pull request merged**